Lib/ctypes/test/test_pep3118.py

```diff
@@ -86,6 +86,20 @@ class PackedPoint(Structure):
     _pack_ = 2
     _fields_ = [("x", c_long), ("y", c_long)]
 
+class PointMidPad(Structure):
+    _fields_ = [("x", c_byte), ("y", c_uint32)]
+
+class PackedPointMidPad(Structure):
+    _pack_ = 2
+    _fields_ = [("x", c_byte), ("y", c_uint32)]
+
+class PointEndPad(Structure):
+    _fields_ = [("x", c_uint32), ("y", c_byte)]
+
+class PackedPointEndPad(Structure):
+    _pack_ = 2
+    _fields_ = [("x", c_uint32), ("y", c_byte)]
```

Comment:
I think I'm right in saying that using `c_uint32` ensure that:

1. the tests can just use `L`
2. the alignment will be 4